### ISSUE TITLE
安卓9无法使用脚本

### ISSUE THREAD

**Issue by a790186435** [TonyJiangWJ/Ant-Forest#238]
### 验证步骤

- [x] 我已提供清晰的问题描述以及复现步骤和日志。
- [x] 我已阅读并遵循了项目 [README](https://github.com/TonyJiangWJ/Ant-Forest/blob/master/README.md) 的指引，已经正确阅读常见问题中所描述的内容。
- [x] 我已搜索现有 [Issues](https://github.com/TonyJiangWJ/Ant-Forest/issues?q=is%3Aissue)，确认没有重复问题。
- [ ] 功能疑问请移步 [Discussions](https://github.com/TonyJiangWJ/Ant-Forest/discussions) 或者 [论坛](https://autoscripts.flarum.cloud/)。

### 描述

手机：荣耀v20，安卓9
运行脚本时提示Failed resolution of: Landroid/view/accessibility/AccessibilityNodeInfo$ExtraRenderingInfo
问了ai好像说是只能在安卓11以上运行

### 重现方式

运行跟截图有关的脚本都提示

### 日志/截图

02:26:42.334/E: Failed resolution of: Landroid/view/accessibility/AccessibilityNodeInfo$ExtraRenderingInfo;

java.lang.NoClassDefFoundError: Failed resolution of: Landroid/view/accessibility/AccessibilityNodeInfo$ExtraRenderingInfo;
	at java.lang.reflect.Executable.getMethodReturnTypeInternal(Native Method)
	at java.lang.reflect.Method.getReturnType(Method.java:148)
	at org.mozilla.javascript.JavaMembers.extractGetMethod(JavaMembers.java:697)
	at org.mozilla.javascript.JavaMembers.findGetter(JavaMembers.java:684)
	at org.mozilla.javascript.JavaMembers.reflect(JavaMembers.java:576)
	at org.mozilla.javascript.JavaMembers.<init>(JavaMembers.java:54)
	at org.mozilla.javascript.JavaMembers.createJavaMembers(JavaMembers.java:827)
	at org.mozilla.javascript.JavaMembers.lookupClass(JavaMembers.java:786)
	at org.mozilla.javascript.NativeJavaClass.initMembers(NativeJavaClass.java:50)
	at org.mozilla.javascript.NativeJavaObject.<init>(NativeJavaObject.java:53)
	at org.mozilla.javascript.NativeJavaClass.<init>(NativeJavaClass.java:44)
	at org.mozilla.javascript.NativeJavaClass.<init>(NativeJavaClass.java:40)
	at org.mozilla.javascript.WrapFactory.wrapJavaClass(WrapFactory.java:136)
	at org.mozilla.javascript.NativeJavaPackage.getPkgProperty(NativeJavaPackage.java:134)
	at org.mozilla.javascript.NativeJavaPackage.get(NativeJavaPackage.java:84)
	at org.mozilla.javascript.ScriptableObject.getProperty(ScriptableObject.java:2023)
	at org.mozilla.javascript.ScriptRuntime.getObjectProp(ScriptRuntime.java:1672)
	at org.mozilla.javascript.ScriptRuntime.getObjectProp(ScriptRuntime.java:1667)
	at org.mozilla.javascript.Interpreter.interpretLoop(Interpreter.java:1559)
	at org.mozilla.javascript.Interpreter.interpret(Interpreter.java:1061)
	at org.mozilla.javascript.InterpretedFunction.call(InterpretedFunction.java:90)
	at org.mozilla.javascript.AccessorSlot$FunctionGetter.getValue(AccessorSlot.java:128)
	at org.mozilla.javascript.AccessorSlot.getValue(AccessorSlot.java:65)
	at org.mozilla.javascript.ScriptableObject.get(ScriptableObject.java:241)
	at org.mozilla.javascript.IdScriptableObject.get(IdScriptableObject.java:367)
	at org.mozilla.javascript.ScriptableObject.getProperty(ScriptableObject.java:2023)
	at org.mozilla.javascript.ScriptRuntime.getObjectProp(ScriptRuntime.java:1672)
	at org.mozilla.javascript.ScriptRuntime.getObjectProp(ScriptRuntime.java:1667)
	at org.mozilla.javascript.Interpreter.interpretLoop(Interpreter.java:1559)
	at org.mozilla.javascript.Interpreter.interpret(Interpreter.java:1061)
	at org.mozilla.javascript.InterpretedFunction.call(InterpretedFunction.java:90)
	at org.mozilla.javascript.ContextFactory.doTopCall(ContextFactory.java:380)
	at org.mozilla.javascript.ScriptRuntime.doTopCall(ScriptRuntime.java:3869)
	at org.mozilla.javascript.InterpretedFunction.exec(InterpretedFunction.java:103)
	at com.stardust.autojs.engine.RhinoJavaScriptEngine.doExecution(RhinoJavaScriptEngine.kt:99)
	at com.stardust.autojs.engine.JavaScriptEngine.execute(JavaScriptEngine.java:22)
	at com.stardust.autojs.engine.LoopBasedJavaScriptEngine.access$001(LoopBasedJavaScriptEngine.java:20)
	at com.stardust.autojs.engine.LoopBasedJavaScriptEngine.lambda$execute$0$com-stardust-autojs-engine-LoopBasedJavaScriptEngine(LoopBasedJavaScriptEngine.java:45)
	at com.stardust.autojs.engine.LoopBasedJavaScriptEngine$$ExternalSyntheticLambda0.run(Unknown Source:6)
	at android.os.Handler.handleCallback(Handler.java:907)
	at android.os.Handler.dispatchMessage(Handler.java:105)
	at android.os.Looper.loop(Looper.java:216)
	at com.stardust.autojs.engine.LoopBasedJavaScriptEngine.execute(LoopBasedJavaScriptEngine.java:64)
	at com.stardust.autojs.engine.LoopBasedJavaScriptEngine.execute(LoopBasedJavaScriptEngine.java:37)
	at com.stardust.autojs.execution.LoopedBasedJavaScriptExecution.doExecution(LoopedBasedJavaScriptExecution.java:42)
	at com.stardust.autojs.execution.RunnableScriptExecution.execute(RunnableScriptExecution.java:56)
	at com.stardust.autojs.execution.RunnableScriptExecution.execute(RunnableScriptExecution.java:50)
	at com.stardust.autojs.execution.RunnableScriptExecution.run(RunnableScriptExecution.java:36)
	at java.lang.Thread.run(Thread.java:784)
	at com.stardust.lang.ThreadCompat.run(ThreadCompat.java:78)
Caused by: java.lang.ClassNotFoundException: Didn't find class "android.view.accessibility.AccessibilityNodeInfo$ExtraRenderingInfo" on path: DexPathList[[zip file "/data/app/com.litiaotiao.app-UIkWcMjlDYo4A0dEXhqzBg==/base.apk"],nativeLibraryDirectories=[/data/app/com.litiaotiao.app-UIkWcMjlDYo4A0dEXhqzBg==/lib/arm, /data/app/com.litiaotiao.app-UIkWcMjlDYo4A0dEXhqzBg==/base.apk!/lib/armeabi-v7a, /system/lib, /product/lib]]
	at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:134)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:379)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
	... 50 more

![Image](https://github.com/user-attachments/assets/5f2a9b36-af0d-4384-b205-4b9d27ddb595)

[log-verboses\[1\].log](https://github.com/user-attachments/files/22453847/log-verboses.1.log)
